### ISSUE TITLE
Some external repos can't be loaded due to bad BUILD.bazel file

### ISSUE THREAD

**Issue by katre** [bazelbuild/bazel#2252]
Sample error:
```
WARNING: /usr/local/google/home/jcater/.cache/bazel/_bazel_jcater/0b1f75abcda7b0da0fb692fc98a70dc5/external/io_bazel_skydoc/skylark/skylark.bzl:166:15: Variables HOST_CFG and DATA_CFG are deprecated in favor of strings "host" and "data" correspondingly.
ERROR: /usr/local/google/home/jcater/.cache/bazel/_bazel_jcater/0b1f75abcda7b0da0fb692fc98a70dc5/external/io_bazel_rules_go/go/toolchain/BUILD:10:1: no such package '@io_bazel_rules_go_toolchain//': Not a file: /usr/local/google/home/jcater/.cache/bazel/_bazel_jcater/0b1f75abcda7b0da0fb692fc98a70dc5/external/golang_linux_amd64/BUILD and referenced by '@io_bazel_rules_go//go/toolchain:go_tool'.
ERROR: /usr/local/google/home/jcater/.cache/bazel/_bazel_jcater/0b1f75abcda7b0da0fb692fc98a70dc5/external/io_bazel_rules_go/go/toolchain/BUILD:15:1: no such package '@io_bazel_rules_go_toolchain//': Not a file: /usr/local/google/home/jcater/.cache/bazel/_bazel_jcater/0b1f75abcda7b0da0fb692fc98a70dc5/external/golang_linux_amd64/BUILD and referenced by '@io_bazel_rules_go//go/toolchain:go_include'.
ERROR: /usr/local/google/home/jcater/.cache/bazel/_bazel_jcater/0b1f75abcda7b0da0fb692fc98a70dc5/external/io_bazel_rules_go/go/toolchain/BUILD:5:1: no such package '@io_bazel_rules_go_toolchain//': Not a file: /usr/local/google/home/jcater/.cache/bazel/_bazel_jcater/0b1f75abcda7b0da0fb692fc98a70dc5/external/golang_linux_amd64/BUILD and referenced by '@io_bazel_rules_go//go/toolchain:toolchain'.
```

**Comment by katre** [comment]
This is caused by 2f31d60d, will roll it back.

**Comment by katre** [comment]
All breakages are due to rules_go, fixing that instead.

**Comment by katre** [comment]
Fixed in the immediate case by reverting fa3f55a8785c282.
